src/Microsoft.Fast.Components.FluentUI/Microsoft.Fast.Components.FluentUI.csproj

```diff
@@ -14,8 +14,8 @@
     <Title>Microsoft Fluent UI Components Library for Blazor on FAST</Title>
     <Description>Authored by the Microsoft FAST team.</Description>
     <SignAssembly>false</SignAssembly>
-    <PackageVersion>0.0.2</PackageVersion>
-    <ReleaseVersion>0.0.2</ReleaseVersion>
+    <PackageVersion>1.0.0</PackageVersion>
```

Comment:
Review previous PR comments.

Reply:
the preview PR was intended for ADO. since ADO was defaulting to version 1.0.0